src/Elewant/Webapp/Infrastructure/Twig/ComponentsExtension.php

```diff
@@ -13,6 +13,7 @@
 
 final class ComponentsExtension extends AbstractExtension
 {
+    /** @return TwigFunction[] */
```

Comment:
What's with the short notation on methods also? I can sort-of accept it for properties but this is confusing.